packages/xdl/src/Env.ts

```diff
@@ -0,0 +1,15 @@
+import os from 'os';
+
+import getenv from 'getenv';
+
+export function home(): string {
```

Comment:
Return types are usually not required, TypeScript can figure them out itself.

Reply:
Even when the type inference can usually figure out the return type, I don't consider explicit annotations to be an error, particularly for exported functions. They document the intended return type and make it easier to see if a change to the code would change the public API of a module.

Reply:
I don't consider this to be an error either. I was just curious if you're aware of that.